yammbs/torsion/inputs.py

```diff
@@ -86,6 +90,7 @@ def from_qcsubmit_collection(
                         explicit_hydrogens=True,
                     ),
                     dihedral_indices=record.specification.keywords.dihedrals[0],  # assuming this is only ever 1-len?
```

Comment:
This can actually be 2D (e.g. the protein torsion drives), but let's keep things simple for now!